.travis.yml

```diff
@@ -6,20 +6,24 @@ go:
 services:
   - rabbitmq
 
+sudo: required
```

Comment:
My reading of past pull requests was that we tried to add sudo: false, but there was some statsd behavior that broke it.

I think in the meantime, Travis quietly switched defaults, so it now defaults to container builds? Bill and I were getting errors messages from Travis saying we needed to set sudo: required, which is why we added that.

I would definitely like to get us to a point where we can run in a container. AFAIK that means resolving two issues: the need to install dependencies with apt-get for the letsencrypt-preview integration tests; and statsd's behavior (not exactly sure what the issue was here).